docker-compose.yml

```diff
@@ -29,37 +29,22 @@ services:
       AGENT_ENDPOINT: https://didcomm.demo.animo.id
       AGENT_PUBLIC_DID_SEED: ${AGENT_PUBLIC_DID_SEED}
 
-    volumes:
-      - indy_client:/root/.indy_client
-
-    configs:
-      - source: afj_config
-        target: /config.json
-
     networks:
       - traefik
 
     ports:
       - '5000'
       - '5001'
 
-    command: --config /config.json
-
   client:
     image: ghcr.io/animo/animo-demo/client:release
     deploy:
       placement:
         constraints:
           - node.role == worker
       labels:
-        # Enable traefik so the container can be exposed to the outside world
         traefik.enable: 'true'
-
         traefik.http.routers.client-demo.rule: Host(`demo.animo.id`)
-        traefik.http.routers.client-demo.entrypoints: web-secure
-        traefik.http.routers.client-demo.tls.certresolver: zerossl
-        traefik.http.routers.client-demo.service: client-demo-service
```

Comment:
These are needed @janrtvld, just the middleware is not needed!

Reply:
Sorry should have been more clear

Reply:
Ye just fixed it thanks